docs/UserGuide.md

```diff
@@ -75,78 +65,70 @@ Format: `help`
 
 ### Adding a person: `add`
 
-Adds a person to the address book.
+Adds a client to ClientBook.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
+Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [i/POLICY_ID] …​`
```

Comment:
Would it be better to display all the possible parameters?

Reply:
Added tags back in